bunsen-avro/src/main/java/com/cerner/bunsen/avro/converters/DefinitionToAvroVisitor.java

```diff
@@ -34,6 +34,7 @@
 import org.apache.avro.generic.GenericData.Record;
 import org.apache.avro.generic.IndexedRecord;
 import org.apache.avro.specific.SpecificData;
+import org.apache.avro.specific.SpecificRecord;
```

Comment:
Is this a needed import?

Reply:
No, removed in 2a1ef489bf2a1df2cde5950bcb0ea42c2350bc1b